doctor.py

```diff
@@ -67,25 +67,33 @@ def main(context: Dict[str, str], argv: Sequence[str] | None = None) -> int:
         if args.check_only:
             return 1
 
-    print("\nAttempting to run fixes for the checks that failed...")
-    futures = {}
-    results = {}
+    print("\nThe following problems have been identified:")
```

Comment:
This won't print everything upfront which is confusing. For example:

```
Running checks: foo, foo2
❌ check: foo/Users/josh/.local/share/sentry-devenv/devenv/foo doesn't exist
❌ check: foo2/Users/josh/.local/share/sentry-devenv/devenv/foo2 doesn't exist

The following problems have been identified:
❌ foo
Do you want to attempt to fix foo? (Y/n): n
⏭️ Skipping foo.
❌ foo2
Do you want to attempt to fix foo2? (Y/n): n
⏭️ Skipping foo2.

Checking again...
⏭️ Skipped foo.
⏭️ Skipped foo2.

Looks good to me now!
```

Reply:
Good catch

Reply:
@joshuarli what do you find confusing about this? It seems ok to me.

Side note/future change: It would be nice to indent some of these logs to add some logical grouping.

```
Running checks: foo, foo2
    ❌ check: foo/Users/josh/.local/share/sentry-devenv/devenv/foo doesn't exist
    ❌ check: foo2/Users/josh/.local/share/sentry-devenv/devenv/foo2 doesn't exist


The following problems have been identified:
    ❌ foo
        Do you want to attempt to fix foo? (Y/n): n
        ⏭️ Skipping foo.

    ❌ foo2
        Do you want to attempt to fix foo2? (Y/n): n
        ⏭️ Skipping foo2.


Checking fixes worked as expected...
    ⏭️ Skipped foo.
    ⏭️ Skipped foo2.

Looks good to me now!
```